docs/introduction/Installation.md

```diff
@@ -8,6 +8,85 @@
 
 ## vcpkg
 
+## Compiling from Source - Ubuntu 16.04 LTS (Xenial Xerus) 
+Please run the following commands and read the comments as you go.
+You will notice that we update and upgrade quite frequently, this is to ensure that all dependancies are met at all times. You will also notice that we are installing some libraries from source, there are two reasons for this; firstly in the event that the library is not available through apt-get or secondly, because we want the very latest version (not the version available in apt-get).
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install cmake
+sudo apt-get install zlibc
+sudo apt-get install zlib1g
+sudo apt-get install zlib1g-dev
+sudo apt-get install libssl-dev
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+sudo apt update
+sudo apt-get upgrade
+sudo apt-get install gcc-6 g++-6
+sudo apt update
+sudo apt-get upgrade
+```
+The following steps will set up some environment variables for this session
+```
+export CC=/usr/bin/gcc-6  
+export CXX=/usr/bin/g++-6
+```
+The following steps will install the appropriate crypto libraries for c++
+Download the zip file from https://github.com/weidai11/cryptopp/archive/master.zip
+```
+cd ~
+unzip Downloads/cryptopp-master.zip
+cd cryptopp-master
+make -j 2
+sudo make install
+```
+The following steps will install the latest GCC compiler from source (you will notice that we installed GCC 6 using apt-get, this was a building block to the next step of getting GCC 6.3.0 - the latest compiler available at present).
```

Comment:
Sure thing, shall we proceed with 6.3.0 for now and put in a documentation issue/request (which I will perform). I wouldn't mind conducting a full re-install and carrying out some testing as part of the move to GCC 7.1. 
I initially upgraded the documentation to 6.3.0 as I was having issues when using older GCC versions. Seems to be working great with 6.3.0 for the immediate future.